homeassistant/components/asuswrt/router.py

```diff
@@ -57,13 +136,84 @@ async def setup(self) -> None:
         if not self._api.is_connected:
             raise ConfigEntryNotReady
 
+        # Load tracked entities from registry
+        entity_registry = await self.hass.helpers.entity_registry.async_get_registry()
+        track_entities = (
```

Comment:
Please name this variable something with entries instead of entities. We reserve the latter for `Entity` instances.